v-next/hardhat-utils/package.json

```diff
@@ -26,7 +26,8 @@
     "./number": "./dist/src/number.js",
     "./package": "./dist/src/package.js",
     "./request": "./dist/src/request.js",
-    "./string": "./dist/src/string.js"
+    "./string": "./dist/src/string.js",
+    "./sub-process": "./dist/src/subprocess.js"
```

Comment:
I think you forgot to rename it here, it's still `sub-process`

Reply:
Done in [this commit](https://github.com/NomicFoundation/hardhat/pull/5471/commits/b3ed594afe2438f7585cb8834b7359a829e973f4) with other improvements